packages/frontend-2/components/dashboards/share/Dialog.vue

```diff
@@ -0,0 +1,152 @@
+<template>
```

Comment:
@gjedlicska Kept it simple for now, as we are not sure about the other sharing dialogs yet I didnt bother yet with turning this into a reusable singleton, if there is a pattern we can do that later 

<img width="561" height="341" alt="Image" src="https://github.com/user-attachments/assets/cf1896a6-e320-4016-b84d-1fcb7ed2ce28" />